docs/verify/simulate-with-primitives.mdx

```diff
@@ -0,0 +1,451 @@
+---
+title: Simulate with Qiskit primitives
+description: Simulate with Qiskit reference primitives. How to compute an expectation value with the Estimator primitive, and how to compute circuit output probabilities with the Sampler primitive
+---
+
+# Simulate with Qiskit primitives
+
+The reference primitives in Qiskit can perform local statevector simulations, which is useful for quickly prototyping algorithms. The `Estimator` primitive can compute an expectation value, and the `Sampler` primitive can compute circuit output probabilities. For import instructions, see the topic [Where should I import the primitive from?](../run/primitives#import-primitives)
```

Comment:
Feel free to disagree.

```suggestion
The reference primitives in Qiskit can perform local statevector simulations, which is useful for quickly prototyping algorithms. 

The `Estimator` primitive can compute an expectation value, and the `Sampler` primitive can compute circuit output probabilities. For import instructions, see the topic [Where should I import the primitive from?](../run/primitives#import-primitives)
```

Reply:
I would want to also add another carriage return before the last sentence about import instructions, if we add one here.